products/sle/main.pm

```diff
@@ -646,6 +646,28 @@ sub prepare_target {
     }
 }
 
+sub is_baremetal_test {
+    return get_var('IBTESTS');
```

Comment:
Maybe we could be more generic and create a global variable for BM. 
NFV tests are also for BM but can't use this function. What about reading if variable BAREMETAL=1 is set and return true if so (or something similar).

Reply:
I thought about this. However, setting up these tests already involves correctly setting quite some Variables. This is why I decided to do it like it is already done for the kernel tests. I'm not sure if another variable would be helpful.